salt/buildbot/init.sls

```diff
@@ -70,10 +69,12 @@ update-master:
   cmd.run:
     - runas: buildbot
     - cwd: /srv/buildbot
-    - name: make update-master
+    - name: make update-master PYTHON_VERSION=3.13
     - require:
+      - pkg: buildbot-deps
       - git: /srv/buildbot
     - onchanges:
+      - pkg: buildbot-deps
```

Comment:
I'm unsure if this is specified correctly, but my goal is to make sure this is run after this is merged (and after any future updates).  Do both `require` and `onchanges` need to be set?